### ISSUE TITLE
influxdb - Axes tab to set units is missing

### ISSUE THREAD

**Issue by steverweber** [mtanda/grafana-histogram-panel#34]
Axes tab to set units would be nice to have,

**Comment by rfcohen** [comment]
Would love to be able to set labels on the axes as well as have the y axis be logarithmic.  Thanks.

**Comment by stevenroose** [comment]
Yeah we need this feature too. Is anyone on this? Graphs has an `Axes` tab.
